crates/editor/src/hover_popover.rs

```diff
@@ -525,7 +525,7 @@ async fn parse_blocks(
                     font_family: Some(buffer_font_family),
                     ..Default::default()
                 },
-                rule_color: Color::Muted.color(cx),
+                rule_color: cx.theme().colors().border,
```

Comment:
```suggestion
                rule_color: cx.theme().colors().border_variant,
```

The after screenshot is looking good to me, but if you want to experiment with a slightly darker color, try this out!

Reply:
If you’re happy with matching the git popover then let’s just go with this?